src/blocks/battery.rs

```diff
@@ -10,11 +10,10 @@
 //! `device` | The device in `/sys/class/power_supply/` to read from. When using UPower, this can also be `"DisplayDevice"`. Regular expressions can be used. | Any battery device
 //! `driver` | One of `"sysfs"`, `"apc_ups"`, or `"upower"` | `"sysfs"`
 //! `interval` | Update interval, in seconds. Only relevant for `driver = "sysfs"` \|\| "apc_ups"`. | `10`
-//! `format` | A string to customise the output of this block. See below for available placeholders. | <code>"$percentage&vert;"</code>
-//! `full_format` | Same as `format` but for when the battery is full | `""`
-//! `empty_format` | Same as `format` but for when the battery is empty | `""`
-//! `hide_missing` | Completely hide this block if the battery cannot be found. | `false`
-//! `hide_full` | Hide the block if battery is full | `false`
+//! `format` | A string to customise the output of this block. See below for available placeholders. | <code>" $icon $percentage&vert; "</code>
+//! `full_format` | Same as `format` but for when the battery is full | `" $icon "`
+//! `empty_format` | Same as `format` but for when the battery is empty | `" $icon "`
+//! `format_missing` | Same as `format` if the battery cannot be found. | `" $icon × "`
```

Comment:
The naming seems inconsistent, we have `{full,empty}_format` so `missing_format` might be better.

Reply:
We also have `format_alt` :smile:  :see_no_evil: I'll change it to `missing_format`